### ISSUE TITLE
Первый этап ПФУ

### ISSUE THREAD

**Issue by iprst** [audetv/cohesion#2]
Этот репозиторий предлагаю использовать в качестве отправной точки для размещения материалов, репозитория для которых пока не существует или его вообще не планируется. 

Здесь же можно открывать issue общего назначения, тематика которых не вполне ясна на момент возкникновения идеи или вопроса, то есть пользуемся [cohesion](https://github.com/terratensor/cohesion/issues) как первым этапом ПФУ.

**Comment by iprst** [comment]
Например. 

Я пытался найти какое-то сочетание слов для названия организации, и в результате был дважды забанен гитхабом за частоту проверок свободных названий. Список был большой и достаточно наивный, в качесте опыта опишу кратко для общего знания. 

Сначала я подумал: у нас названия svodd и geomatrix очень хорошие, притом по-разному, первое отправляет к русскому своду, а также прямо обозначает наш глобальный процесс СВОДД, однако неподготовленный товарищ этого скорее всего не прочитает без пояснения. Геоматрица также имеет подобную проблему, но всё же позволяет что-то уловить о методе чисто из частей слова, однако в качестве названия организации на мой взгляд не подходит. 

Я открыл вики-коб поискать управленческие термины, которые либо можно набрать латиницей как svodd, либо изначально греческие или латинские, как геоматрица. В качестве первого сразу вылезла mera — но на удивление это слово очень широко используется на гитхабе, оно зарегистрировано само по себе и также неоднократно включено в состав сложных слов.

Также попался ИНВОУ, который по факту не переводим ни на английский, ни просто в латиницу — в английском языке нет слова управление в том смысле, в котором оно используется в КОБ. Но из иерархически наивысшего всеобъемлющего управления нашёлся некоторый набор интересных слов, которые можно было бы использовать как отправные точки или части для соединения со смежными понятиями. Вот он, сильно не смейтесь:

```
hierarchy
graded 
ranked
supreme
hyper
ultimate
universal
integral
ruler
```
Добавляем сюда же меру. Раз такое дело, добавим также и информацию, раз у нас тут во всю IT — будет data. Почему бы не добавить и собор, добавим и его. Также поскольку мы понимаем, что у нас не просто абстрактный поисковик, а скорее связанный с нашими глобально-планетарными событиями, то есть с геоматрицей, были добавлены и гео с матрицей. Подумав ещё минутку были найдены синонимы как для приставки гео, так и для матрицы:
```
mera
sobor
geo
globe
global
terra
matrix
tensor
```
Любопытно, что по сути «многомерная матрица» это широко используемый в компьютерных делах термин tensor, который в обратную сторону читается как rosnet — как вы понимаете, оба слова уже заняты, занято также и слово tensa, сокращение от полного тензора. Всё занято, в общем. Поэтому далее были многочисленные попытки что-то соединить из полученного набора:
```
unimatrix
ultimatrix
suprematrix
hypertensor
geotensor
terramera
ultimera
meradata
hypermera
hyperdata
hyperdatum
geotensor
```
Были и другие приставки и корни в том же духе, но все проверки говорили что имя занято. Свободными были только несколько вариантов: 
```
globerank
georank
meratensor
terratensor
meramatrix
datamera
tensorglobe
```
Из этого набора свободны все имена на гитхабе, однако гуглопочту можно было зарегистрировать только на _meratensor_, _terratensor_ или _tensorglobe_. По звучанию _terratensor_ наиболее звучный, более того является т.н. гиперонимом для геоматрицы, обобщающим синонимом: гео и терра это одно и то же, а тензор это «расширение» матрицы. Кроме того, terratensor.ru свободен. 

Так и получился: 

![terratensor](https://github.com/terratensor/cohesion/assets/129882753/886f53ad-b532-4f67-8470-3870839aec13) ![ttlogo](https://github.com/terratensor/cohesion/assets/129882753/98e1620b-76f4-4782-a54b-0c43bcc51685)

**Comment by iprst** [comment]
В дальнейшем можно переименовать или создать новую организацию с более подходящим названием, но на текущий момент другого звучного и вместе с тем свободного названия для организации у меня найти не получилось, поэтому даже не было смысла создавать голосовалку. Если что-то найдёте, предлагайте.

**Comment by audetv** [comment]
Завтра перенесу поиск.

> В дальнейшем можно переименовать или создать новую организацию с более подходящим названием, но на текущий момент другого звучного и вместе с тем свободного названия для организации у меня найти не получилось, поэтому даже не было смысла создавать голосовалку. Если что-то найдёте, предлагайте.

Мне нравится это именование и история появления. 

Стартовый список выглядит как названия каких-то игр, но из него получилось отличное название
```
hierarchy
graded 
ranked
supreme
hyper
ultimate
universal
integral
ruler
```

**Comment by iprst** [comment]
> Завтра перенесу поиск.

Есть мысль, что fct-search при переносе имеет смысл переименовать в svodd, поскольку судя по всему поисковик будет развиваться в эту линию и далее, домен уже есть, и так будет кажется более всего наглядно. Если это конечно не мешает технической стороне дела. 

> Стартовый список выглядит как названия каких-то игр, но из него получилось отличное название

Все слова этого списка по сути синонимичные варианты расшифровки ИНВОУ, но если их написать одной строкой это может быть название Главаря Очень Серьёзной Организации в комиксах: 

```
hierarchy graded supreme ranked hyper ultimate universal integral ruler
иерархически градуированный превосходящий ранговый гиперфинальный вселенский конечный правитель
```

**Comment by audetv** [comment]
> Есть мысль, что fct-search при переносе имеет смысл переименовать в svodd, поскольку судя по всему поисковик будет развиваться в эту линию и далее, домен уже есть, и так будет кажется более всего наглядно. Если это конечно не мешает технической стороне дела.

Да, так и сделаем. Проверю все завтра, без спешки перенесу.

> ```
> hierarchy graded supreme ranked hyper ultimate universal integral ruler
> иерархически градуированный превосходящий ранговый гиперфинальный вселенский конечный правитель
> ```

Да, такие слова, а получился комикс)

**Comment by audetv** [comment]
> Этот репозиторий предлагаю использовать в качестве отправной точки для размещения материалов, репозитория для которых пока не существует или его вообще не планируется.

Можно включить дискуссию. Вот что предлагается. И выбрать этот репозиторий для отправной точки.

<details><summary>Turn on discussions</summary>
<p>

![2023-06-09_22-16-54](https://github.com/terratensor/cohesion/assets/10896447/8ceae676-eadb-485c-bc86-f8167d32e6e0)

</p>
</details>

**Comment by audetv** [comment]
> Кроме того, terratensor.ru свободен.

Надо зарегистрировать домен.

**Comment by iprst** [comment]
> Надо зарегистрировать домен.

Да. Я смогу только в будний день. Но если хотите, можете оформить на себя.

**Comment by audetv** [comment]
> > Надо зарегистрировать домен.
> 
> Да. Я смогу только в будний день. Но если хотите, можете оформить на себя.

Хорошо. В будний день зарегистрируете.

**Comment by iprst** [comment]
> Хорошо. В будний день зарегистрируете.

Договорились.

**Comment by audetv** [comment]
Надо, видимо, заглавное изображение сделать на тёмном фоне, иначе это выглядит блекло на светлом фоне.
у меня gthub на десктопе в светлом режиме, не смог перестроиться, и у других товарищей может быть также.
На темной подложке выглядит лучше.

![2023-06-10_11-15-13](https://github.com/terratensor/cohesion/assets/10896447/999357b7-ddef-4547-b73a-7851ea4facda)

Типа как вот это
 https://user-images.githubusercontent.com/129882753/244776075-886f53ad-b532-4f67-8470-3870839aec13.png

UPD: Интересно, не знал, если нажать значок редактировать на заглавном экране , то там код и указанно 2 режима dark и light
<details><summary>terratensor/.github is a special repository: this README.md will appear on your public organization profile, visible to anyone.</summary>
<p>

![image](https://github.com/terratensor/cohesion/assets/10896447/65cec7d3-2e35-49c1-bc72-0f5d0d53daf8)

</p>
</details> 

Получается, что можно разные изображения указать для разных режимов.

**Comment by iprst** [comment]
> Интересно, не знал, если нажать значок редактировать на заглавном экране , то там код и указанно 2 режима dark и light

Так это я их вручную ввёл, по умолчанию там нет никакого кода. Заодно откомментировал о том, что смена режимов работает только для статично выбранной темы, тёмной или светлой — при динамическом выборе в зависимости от настроек системы это не работает, или работает как-то неправильно. 

Светлое и тёмное изображение это два разных изображения, отрендеренных в редакторе по отдельности. Ночью показалось что достаточно контрастно, но можно прибавить.

**Comment by iprst** [comment]
> Надо, видимо, заглавное изображение сделать на тёмном фоне, иначе это выглядит блекло на светлом фоне.

Увеличил контрастность светлой темы, проверьте у себя. Можно в принципе ещё дальше увеличивать (или уменьшать), это векторный проект в Inkscape. 

> Получается, что можно разные изображения указать для разных режимов.

Раньше на гитхабе была более простая схема динамической подмены — к хвосту имени файла приписывался хэштег, для светлой или тёмной темы. Но затем эту конструкцию упразднили и [сделали более строгую](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#specifying-the-theme-an-image-is-shown-to), видимо по канонам текущего html/css. Подозреваю потому, что появилось несколько светлых и тёмных тем, и набор стал диамическим (флаг отдаёт либо браузер, либо операционка, этого я не знаю). Так вот в динамическом режиме судя по всему у всех проблемы с этим функционалом, [нашлось три issue](https://github.com/github/markup/issues?q=prefers-color-scheme) в официальном репозитории markdown гитхаба — картинка отдаётся чехардой, у кого одно, у кого другое. В статически выбранной теме всё в порядке. 

А вот аватар сделать отдельный для тёмной и светлой темы нельзя, могло быть интересным.

**Comment by audetv** [comment]
> Увеличил контрастность светлой темы, проверьте у себя. Можно в принципе ещё дальше увеличивать (или уменьшать), это векторный проект в Inkscape.

Так да, текст более контрастен. 

> Так это я их вручную ввёл, по умолчанию там нет никакого кода. Заодно откомментировал о том, что смена режимов работает только для статично выбранной темы, тёмной или светлой — при динамическом выборе в зависимости от настроек системы это не работает, или работает как-то неправильно.

Да, я написал сообщение, а потом понял, что это вы сделали, почему-то вначале показалось, что это что-то из стандартного функционала, шаблона, даже на коммиты внимание не обратил), тот момент, когда взглянул на то, с чем знаком по факту много лет, глазами новичка, забавно))

**Comment by iprst** [comment]
> вначале показалось, что это что-то из стандартного функционала

Да, этот функционал настолько логичен, что как-бы ожидается, что он должен существовать по умолчанию изначально. 

> Так да, текст более контрастен.

Можно менять по необходимости. Картинка в основном чисто для номинального оформления, если не мешает, можно оставлять.

ПС. Вряд ли кто-то догадается, что на [аватаре terratensor](https://avatars.githubusercontent.com/u/136109000) скриншот из Google Earth. Я бы не догадался. 

![googlobus](https://github.com/terratensor/cohesion/assets/129882753/de5a49a3-e652-4d2e-b047-db68c9533e9f)

**Comment by audetv** [comment]
Какой прекрасный вопрос на главной, аж, руки зачесались, сижу довольный:

> Здравствуйте команда ФКТ и форумчане! Валерий Викторович часто упоминает про бендероцев, которые во времена бегства Мазепы в Бендеры, с остатками своего войска устраивали набеги на близлежащие села и набирали пленных, пользуясь доверительным к ним отношением местных. Подскажите кто нибудь источники, где почитать о событиях тех времен, также где почитать подробней о действиях лагерей Талергоф и Терезин и в принципе про украинство? Хотелось бы иметь какие нибудь факты про "мазепинских бендеровцев" и "работу" концентрационных лагерей, если вдруг выйдет спор с кем нибудь.

https://xn----8sba0bbi0cdm.xn--p1ai/qa/question/view-48789#:~:text=07:54%2019.07.2023

B tсть ответы на https://lib.svodd.ru

раз:  [Мазепы в Бендеры](https://lib.svodd.ru/?search%5Bquery%5D=+%D0%9C%D0%B0%D0%B7%D0%B5%D0%BF%D1%8B+%D0%B2+%D0%91%D0%B5%D0%BD%D0%B4%D0%B5%D1%80%D1%8B&search%5Bmatching%5D=&search%5Bmatching%5D=query_string)
два: [Талергоф и Терезин](https://lib.svodd.ru/?search%5Bquery%5D=%D0%A2%D0%B0%D0%BB%D0%B5%D1%80%D0%B3%D0%BE%D1%84+%D0%B8+%D0%A2%D0%B5%D1%80%D0%B5%D0%B7%D0%B8%D0%BD&search%5Bmatching%5D=&search%5Bmatching%5D=query_string)

И это просто ctrl+c ctrl+v. Даже выдумывать не надо как спросить. 

Квитанция — мы на правильном пути.

**Comment by iprst** [comment]
> Квитанция — мы на правильном пути.

У меня на этот счёт нет никаких сомнений. 

Вопрос огонь, но что смешнее, почти все вопросы с ФКТ можно сбросить в поиск, как минимум через надстройку с опциями, и получить чёткие результаты. 

Есть такой сценарий — при забросе длинного текста в строку поиска в надстройке, выдавать результаты с указанием «наибольшего сочетания слов», то есть отсеивать не найденные слова из запроса и показывать результат с подзаголовком из сочетания найденных слов. Например, если запрос «собака картошка андерсен лыжи самурай» выдаёт много результатов, когда исключены некоторые слова — есть результаты по «собака андерсен лыжи» и результаты по «картошка самурай», и это можно обозначить в оформлении. Так будет проще ориентироваться.

**Comment by iprst** [comment]
> Какой прекрасный вопрос на главной, аж, руки зачесались, сижу довольный:

Этот опыт можно описать в теме. Прямо взять и на пальцах показать, как это работает. Типа:
1. Копируем текст вопроса с главной Валерию Викторовичу;
2. Отправляем скопированное в поисковую строку [lib.svodd.ru](https://lib.svodd.ru/)
3. Отбираем подходящие результаты;
4. Отвечаем лучше Валерия Викторовича. 

Таким образом можно осуществлять управление главной — отвечая на вопрос по теме «бесконечными» цитатами. Можно сделать бота в телеграме.

**Comment by audetv** [comment]
> > Квитанция — мы на правильном пути.
> 
> У меня на этот счёт нет никаких сомнений.

Да, у меня тоже уже нет сомнений, и не совсем из-за этой квитанции, это как подтверждение, ЯЖО.
Причем, я написал сообщение, а потом прокрутил комментарии, просматривая «по диагонали», и понял, что я этот вопрос уже видел, в один из тех 2х дней, когда работал над алгоритмом для разбиения/соединения параграфов, но видимо так был погружен, что вообще не связал этот вопрос с тем, что я делал.

А потом я вспомнил про тему квантование и дискретизация, про распространение информации, передатчик/приемник,  про меру восприятия приемника. «Управление — процесс информационный, протекающий согласно мере». Я от одного вопроса получил столько квитанций, и обратной связи в виде эмоций))

> Вопрос огонь, но что смешнее, почти все вопросы с ФКТ можно сбросить в поиск, как минимум через надстройку с опциями, и получить чёткие результаты.

Провел эксперимент забросил целиком вопрос в тестовый сервис найти похожий параграф в кабинете. В целом результаты даже есть связанные, но похоже без тонкой настройки, не получить отличные результаты, так что алгоритм, типа вместо Валерия Викторовича отвечает бот в телеграмме, для тех кто любит телегу, нужно проработать, но идея огонь)) оставим для лучших, времен, в том смысле, что накопится опыт, знания и придет время, и можно будет выдать.

> Есть такой сценарий — при забросе длинного текста в строку поиска в надстройке, выдавать результаты с указанием «наибольшего сочетания слов», то есть отсеивать не найденные слова из запроса и показывать результат с подзаголовком из сочетания найденных слов. Например, если запрос «собака картошка андерсен лыжи самурай» выдаёт много результатов, когда исключены некоторые слова — есть результаты по «собака андерсен лыжи» и результаты по «картошка самурай», и это можно обозначить в оформлении. Так будет проще ориентироваться.

Про это пока осмысливаю, пусть крутится фоном.

**Comment by iprst** [comment]
> «Управление — процесс информационный, протекающий согласно мере». Я от одного вопроса получил столько квитанций, и обратной связи в виде эмоций))

Так этот механизм и работает — повысилась мера, прилетел отчёт, сложилось различение. По всей видимости состояние личной меры это не какая-то постоянная величина, а как и тип строя психики некий процесс с колебанием параметров, поэтому надо рассматривать такие «прозрения» как регулярную процедуру, а действуют они кумулятивно, постепенно собирась в мозаику пока не прорвёт на следующий этап её сборки. 

> Провел эксперимент забросил целиком вопрос в тестовый сервис найти похожий параграф в кабинете. В целом результаты даже есть связанные, но похоже без тонкой настройки, не получить отличные результаты, так что алгоритм, типа вместо Валерия Викторовича отвечает бот в телеграмме, для тех кто любит телегу, нужно проработать, но идея огонь)) оставим для лучших, времен, в том смысле, что накопится опыт, знания и придет время, и можно будет выдать.

Да, конечно. Я вообще это писал подразумевая какой-то отложенный этап в будущем, после сбора информации. Сейчас условно «всё работает» и не требуется срочного вмешательства. Я реально думаю надо повернуть внимание на геодезию, а вопросы поисковиков, распознания текстов и прочая будут вариться в фоне, пока что нужно дать этому котелку некую свободу произвола без сознания.

**Comment by audetv** [comment]
Подошел срок продления домена terratensor.
Предлагаю его продлить, несмотря на то, что пока так и не использовали.
Я планировал на нем разместить сервер геоматрицы, который будет отдавать датасеты, и возможно считать геохеш для заданных координат.
Готов оказать помощь в оплате при необходимости.

**Comment by iprst** [comment]
Приветствую. Был в отъезде пару дней. Автоматическое продление не сработало, сейчас вопрос уже решён.
